feed/test.py

```diff
@@ -1,6 +1,65 @@
-"""
-This file demonstrates writing tests using the unittest module. These will pass
-when you run "manage.py test".
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.test.utils import override_settings
+from rest_framework.test import APIRequestFactory
 
-Replace this with more appropriate tests for your application.
-"""
+from feed.views import ChecklistViewSet
+from workflow.models import Checklist, Organization, TolaUser, WorkflowLevel1, WorkflowLevel2
+
+
+class ViewsTest(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user('john', 'lennon@thebeatles.com', 'johnpassword')
+        self.user.is_superuser = True
+        self.user.is_staff = True
+        self.user.save()
+
+        wflvl1 = WorkflowLevel1.objects.create(name='WorkflowLevel1')
+        wflvl2 = WorkflowLevel2.objects.create(name='WorkflowLevel2', workflowlevel1=wflvl1)
+        Checklist.objects.bulk_create([
+            Checklist(name='CheckList_0', workflowlevel2=wflvl2),
+            Checklist(name='CheckList_1', workflowlevel2=wflvl2),
+        ])
+
+    def test_list_checklist_superuser(self):
+        factory = APIRequestFactory()
```

Comment:
As it's generic and used in other test cases, we can put the factory in the setUp function.